.github/workflows/pull-request-labels.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: Add pull request labels
+
+on:
+  pull_request_target:
+    branches:
+      - main
```

Comment:
We do not work with the `main` branch, probably should be excluded

Reply:
done